gbpservice/nfp/orchestrator/drivers/orchestration_driver.py

```diff
@@ -634,6 +640,10 @@ def create_network_function_device(self, device_data,
                                           by=len(interfaces))
             return None
 
+        vendor_data = vendor_data_result.get('result', None)
```

Comment:
Is this result set to None when there is an exception in the function that is registered with job executor?